Localize/lang/strings.cs.json

```
@@ -150,6 +150,7 @@
   "4aaixN": "Prohlídka",
   "4c0uPQ": "Povinný parametr. Název parametru, jehož hodnoty požadujete.",
   "4hi3ks": "Váš tok se aktualizoval.",
+  "4izAMi": "Enter a value to respond with",
```

Fix grammatical issue:

```suggestion
  "4izAMi": "Enter a value to use in the response",
```